packages/metadata-service/src/readers/chitai-gorod.service/index.ts

```diff
@@ -73,6 +79,14 @@ export class ChitaiGorodService {
 
   async getData(isbn: string): Promise<Bundle> {
     const result = await this.readData(isbn);
-    return this.parseData(result);
+
+    const img = await this.digitalSpaceService.createFile({
+      fileExtension: `https://img-gorod.ru${result.info.image_url}`
+        .split('.')
+        .pop(),
+      buffer: result.image,
+    });
+
+    return this.parseData(result.info, img);
```

Comment:
The same here. It's better not to touch `parseData` at all, and use the `cover` prop value to download the image and then replace it with DO Spaces link